### PR TITLE
Do not pip install by default in oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb

### PR DESCRIPTION
**Pull request by j08lue**
- [x] Turn live `pip install` commands into a markdown instruction

Preview here: https://us-ghg-center.github.io/ghgc-docs/pr-preview/pr-53/user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.html

cc @wildintellect for awareness

### CODE REVIEW

**Review by freitagb — CHANGES_REQUESTED** [review]
@j08lue - i think we also talked about moving all of the imports to the first cell. Can we do that as part of this push as well? Then we can establish that as a template for the others.

File: user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb [hunk 1]
```diff
@@ -42,34 +42,12 @@
    "metadata": {},
```
Comment (wildintellect) [inline user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb:42]: <p>The color scheme does not highlight the difference well, hard to find different pixels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/US-GHG-Center/ghgc-docs/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (wildintellect) [inline user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb:42]: <p>remove the print in this block (see comment below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/US-GHG-Center/ghgc-docs/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (wildintellect) [inline user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb:42]: <p>Modify this to not print every call of function. (it's a print line in a different block).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/US-GHG-Center/ghgc-docs/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (freitagb) [reply user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb:42]: <p>yeah, we modified this one to match the color scheme on the webpage. It's hardcoded though which is not ideal.</p>

**Pull request closed without merging**